cpp/include/ray/api/ray_runtime.h

```diff
@@ -0,0 +1,43 @@
+
+#pragma once
+
+#include <cstdint>
+#include <memory>
+#include <msgpack.hpp>
+#include <typeinfo>
+#include <vector>
+
+#include <ray/api/wait_result.h>
+#include <ray/core.h>
+
+namespace ray {
+namespace api {
+
+struct member_function_ptr_holder {
+  uintptr_t value[2];
```

Comment:
define meaningful names for these 2 fields?

Reply:
Just a array to be a holder, no meaning?